src/databricks/labs/remorph/reconcile/recon_config.py

```diff
@@ -172,6 +172,13 @@ class ReconcileOutput:
     mismatch: MismatchOutput | None = None
     missing_in_src: DataFrame | None = None
     missing_in_tgt: DataFrame | None = None
+    threshold_output: ThresholdOutput | None = None
+
+
+@dataclass
+class ThresholdOutput:
+    threshold_df: DataFrame | None
+    threshold_mismatch_count: int = 0
```

Comment:
We can add threshold_mismatch_col also to this. @sundarshankar89 call to implement it now or not.
Cavait: Since threshold comparison is not cached .. we are already hitting the source and target two times (sample and count). If we implement this it will be one more iteration.